matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/members/MembershipService.kt

```diff
@@ -77,7 +77,7 @@ interface MembershipService {
     /**
      * Kick a user from the room
      */
-    suspend fun kick(userId: String, reason: String? = null)
+    suspend fun remove(userId: String, reason: String? = null)
```

Comment:
As it is a change in the SDK api, would be better to keep `kick` as an alias to remove and mark it as deprecated.
You can write in the interface something like:
```
@Deprecated("Use remove now")
suspend fun kick(userId: String, reason: String? = null) = remove(userId, reason)
```